src/main/java/com/rcv/Main.java

```diff
@@ -70,41 +70,18 @@ public static void main(String[] args) {
   // returns: the new ElectionConfig object, or null if there was a problem
   static ElectionConfig loadElectionConfig(String configPath) {
     // config: the new object
-    ElectionConfig config = null;
-
-    // tracks whether we encountered any critical file system errors
-    boolean encounteredFileError = false;
+    ElectionConfig config;
 
     // rawConfig holds the basic election config data parsed from json
     RawElectionConfig rawConfig =
         JsonParser.parseObjectFromFile(configPath, RawElectionConfig.class);
 
     if (rawConfig == null) {
-      System.err.println(String.format("Failed to load config file: %s", configPath));
+      Logger.executionLog(Level.SEVERE, String.format("Failed to load config file: %s", configPath));
+      return null;
     } else {
       config = new ElectionConfig(rawConfig);
-
-      try {
-        FileUtils.createOutputDirectory(config.getOutputDirectory());
-      } catch (UnableToCreateDirectoryException exception) {
-        System.err.println(
-            String.format(
-                "Failed to create output directory: %s\n%s",
-                config.getOutputDirectory(), exception.toString()));
-        encounteredFileError = true;
-      }
-
-      List<String> validationErrors = config.getValidationErrors();
-      if (!validationErrors.isEmpty()) {
-        for (String error : validationErrors) {
-          Logger.severe("Invalid config: %s", error);
-        }
-        config = null;
-      }
-    }
-
-    if (encounteredFileError) {
-      config = null;
+      config.validate();
```

Comment:
Yeah, I realized after the commit I had failed to check the boxes.  Will fix this and the others.  Clearly I have not been making enough commits lately!

As far as the return value, the user should still be able to load into GUI, edit, and save an invalid config file, hence we return the config object here.

Reply:
Hmm, the problem I ran into is that I could still click Tabulate when there was a bad output path, and it would fail and write an audit log to the CWD, which I don't think is desirable behavior. What if, instead of doing validation on load, we do it on save and tabulate actions? I think that might make more sense. This is also making me think that maybe the validation shouldn't write to the audit log until the output path is verified as valid.

Reply:
Yep, Validate should NOT be part of Load or Save.

Validate MUST be the first step of Tabulate (tabulation fails if validation fails).

We should have a "Validate" button on the editor UI, that the user can push at any time to check their work.  (to this end, I think the editor UI needs a console window to show their errors).

Reply:
Agreed, I'll plan to bring a separate validate button in. In this PR, it'd be great if you could move validate from the load method over to the tabulate one.

Reply:
my pleasure :)